.github/workflows/connect-release.yml

```diff
@@ -13,12 +13,23 @@ concurrency:
 
 jobs:
   connect-v9-deploy:
-    # TODO: this is just a test so we should be able to do it from every branch
-    # TODO: for the real thing we probably want to have environment setup
-    # if: startsWith(github.ref, 'refs/heads/release/connect/')
-    uses: ./.github/workflows/template-connect-build-deploy.yml
-    with:
-      awsRoleToAssume: "arn:aws:iam::538326561891:role/gh_actions_trezor_suite_dev_deploy"
-      awsRegion: "eu-central-1"
-      serverHostname: "dev.suite.sldev.cz"
-      serverPath: "connect-release/9"
+    environment: production
```

Comment:
Should be production environment or that one is for suite?

Reply:
I think we can keep production, otherwise credentials wont work and i would need to update it on the aws side.